docs/.vitepress/config.ts

```diff
@@ -3,18 +3,16 @@ import { defineConfig } from 'vitepress'
 // https://vitepress.dev/reference/site-config
 export default defineConfig({
   title: 'Vue Mess Detector',
-  description: 'A static code analysis tool for detecting code smells and best practice violations in **Vue** and **Nuxt** projects.',
+  description:
+    'A static code analysis tool for detecting code smells and best practice violations in **Vue** and **Nuxt** projects.',
```

Comment:
There is still some formatting issue. In this file you have added one line but practically all lines marked as changed.

Can you take a look?